### DISCUSSION TITLE
分区和播放页面好像强制使用新版了

### DISCUSSION THREAD

**Discussion by tiankong4948** [the1812/Bilibili-Evolved#4071] [功能建议 / Ideas]
分区和播放页面好像强制使用新版了
大佬们能不能出点自定义页面内容的功能，类似首页那样
-
或者把简化功能扩展到分区的页面？
现在进去一片图看着都眼晕，而且也看不到简介和弹幕数了，也没有翻页了
目前我只发现[国创]下的几个分区还是用的类似旧分区页的样式，但其实也加了间距和自动缩放
#
播放页是字体和间距都放大了，文字还好点，主要是间距加了非常多，而且还把内容一直拉到了页面的两边
说实话看不过来，现在电脑屏幕都大，但你盯着文字或者具体内容的时候，其实只能看到屏幕中间部分区域的，将内容一直横向填充，充满整个页面，看起来好累，而且一眼也看不过来
感觉在用老年机一样
#
我录的视频https://www.bilibili.com/video/BV1gY4y1R7SL/

**Comment by Geeyun-JY3** [comment]
Bilibili Evolved  不是时光机：https://github.com/the1812/Bilibili-Evolved/discussions/4045#discussioncomment-5205258

**Reply by tiankong4948** [reply]
不用时光机 把首页的那种修改适用到分区页就行了

**Reply by Geeyun-JY3** [reply]
你说的是 `清爽首页` 那种修改？

**Reply by tiankong4948** [reply]
是 那种就行
